dome-headend/dome-udp-server.py

```diff
@@ -0,0 +1,73 @@
+import json
+import logging
+import socket
+
+from BigPacketSender import BigPacketSender
+from DomeConfig import DomeConfig
+
+#############################
+#			CONFIG			#
+#############################
+DOMEJS_HOST = "localhost"
+DOMEJS_PORT = 1444
+
+CONTROLLER_PORT = 10460
+
+ADDRESSABLE_LED_SERVER_HOST = "5202:2234:1046:0000:0000:0000:0000:0001"
+ADDRESSABLE_LED_SERVER_PORT = 4242
+
+log = logging.getLogger('dome-udp-server')
+
+FORMAT_CONS = '%(asctime)s %(name)-12s %(levelname)8s\t%(message)s'
+logging.basicConfig(level=logging.DEBUG, format=FORMAT_CONS)
+
+class DomeHeadendConfig(DomeConfig):
+	domejsSender = None
+	addressableLEDSender = None
+	controllerPort = -1
+
+	def __init__(self, config, domejs_host, domejs_port, addressable_led_server_host, addressable_led_server_port, controller_port):
+		super(DomeHeadendConfig, self).__init__(config)
+		self.domejsSender = BigPacketSender(domejs_host, domejs_port)
+		self.addressableLEDSender = BigPacketSender(addressable_led_server_host, addressable_led_server_port)
+		self.controllerPort = controller_port
+
+	def process_command(self, command):
+		log.debug("%r" % ("LEDS:",))
+		if len(command) == (self.numLeds * 3):
+			log.debug("%r" % ("Valid",))
+			sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+			for currentController in range(self.numControllers):
+				start = int(self.controllers[currentController].start_index)
+				end = int(self.controllers[currentController].start_index + self.controllers[currentController].num_leds)
+				sock.sendto(command[start:end], (self.controllers[currentController].ip, self.controllerPort))
+
+		self.domejsSender.send(command, socket.AF_INET)
+		self.addressableLEDSender.send(bytearray("$") + command, socket.AF_INET6)
```

Comment:
I assume this updates the "IPv6 address for eery LED" state? But I can't find what this is supposed to send to, nothing listens of port 4242. File missing or am I being thick?